cli/tests/integration/info_tests.rs

```diff
@@ -13,7 +14,7 @@ fn info_with_compiled_source() {
   let mut deno = util::deno_cmd()
     .env("DENO_DIR", t.path())
     .current_dir(util::testdata_path())
-    .arg("cache")
+    .arg("run") // run so it gets emitted
```

Comment:
@bartlomieju actually, is this complexity worth it? Should the user even be concerned about if a file is emitted or not? Maybe before this mattered when we used tsc, but now with swc being so fast I don't think the user should care. Thoughts?

Reply:
I guess there will be observable difference between v1.23 and v1.24 - we recommended using `deno cache` to download all deps and transpile them, eg. `deno info` will no longer show path to emitted file after calling `deno cache`, it will only do it after running `deno run`. I'm not sure how impactful that might be

Reply:
That's a good point. I've updated this in the latest commits and reverted this.